target/linux/generic/pending-5.10/484-mtd-spi-nor-add-esmt-f25l16pa.patch

```diff
@@ -0,0 +1,11 @@
+--- a/drivers/mtd/spi-nor/esmt.c
++++ b/drivers/mtd/spi-nor/esmt.c
```

Comment:
If these patches are "pending", please add the full patch as submitted to the kernel list.